varnish/content.md

```diff
@@ -0,0 +1,49 @@
+# What is Varnish?
+
+Varnish is an HTTP accelerator designed for content-heavy dynamic web sites as well as APIs. In contrast to other web accelerators, such as Squid, which began life as a client-side cache, or Apache and nginx, which are primarily origin servers, Varnish was designed as an HTTP accelerator. Varnish is focused exclusively on HTTP, unlike other proxy servers that often support FTP, SMTP and other network protocols.
+
+> [wikipedia.org/wiki/Varnish_(software)](https://en.wikipedia.org/wiki/Varnish_(software))
+
+%%LOGO%%
+
+# How to use this image.
+
+## Basic usage
+
+Create a `default.vcl` file:
+
+```vcl
+vcl 4.0;
+
+backend default {
+  .host = "www.nytimes.com:80";
+}
+```
+
+Then run:
+
+```console
+$ docker run --name my-running-varnish -v /path/to/default.vcl:/usr/local/etc/varnish/default.vcl:ro --tmpfs /usr/local/var/varnish:exec -d %%IMAGE%%
```

Comment:
Should this be `/etc/varnish/default.vcl` like the Dockerfile example and the default `CMD` of the container?

```Dockerfile
CMD ["varnishd", "-F", "-f", "/etc/varnish/default.vcl"]
```

Reply:
it should, good catch. fixed